package/network/config/ltq-vdsl-app/files/10_atm_ptm.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
```

Comment:
> Doing so, a cpe won't be able to come online again if the dsl/tc_layer is changed by the network operator. This might be acceptable for private use, but for devices that are installed on places where no person is on site for the overwhelming time, this could be a real problem. So I would prefer the dirtier but finally safest way with the reboot, as this shouldn't happen to often anyway. :-)

Got your point but still NAK from me. Other devs might have a different opinion on this. I'm the opinion the way to go is getting the dsl subsystem reset working. Everything else is a dirty hack. Sorry.

> So with my approach you have to setup the br2684 in the notify script after loading the atm driver. But if you are right, and no one else then lantiq is using this br2684 init script, it can be dropped at all.

Yes, that was the idea I had in mind. Doing the br2684 stuff (only) in the hotplug script. No idea if it really does make sense to do so and it is kind of out of scope of the PR.

> As the ATM/PTM auto-mode should be also useable for ADSL only devices like danube/ar9/... what about just do it also for them?

I totally missed that there are ptm modules for ar/danube as well. I'm somewhat used to that ADSL is always ATM, which of course isn't right. 

Indeed, it would be required to introduce dsl hotplug scripts for ltq-adsl-app as well to move the br2684 stuff. But most likely I will not touch ltq-adsl-app. I neither have the required documentation nor an ADSL line.